### PR TITLE
docs: add getting started tutorial (using AKS Fleet Manager)

### PR DESCRIPTION
**Pull request by michaelawyu**
### Description of your changes

This PR adds a getting started tutorial which uses Azure Fleet Manager.

I have:

- [x] Run `make reviewable` to ensure this PR is ready for review.

### How has this code been tested

N/A

### Special notes for your reviewer

N/A

### CODE REVIEW

File: docs/getting-started/aks-fleet-manager.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+Azure resource groups allow users to deploy and manage Azure resources in a logical way; to use
+Azure Fleet Manager, you must create a resource group first following the commands below:
+
+```sh
+# You may use different names and/or locations as you see fit.
+#
+# Azure Fleet Manager may not be available in certain locations.
+export RESOURCE_GROUP=fleet-getting-started
+export LOCATION=eastus
```
Comment (britaniar) [inline docs/getting-started/aks-fleet-manager.md:82]: Might want to add a comment that includes examples of the different regions.

File: docs/getting-started/README.md [hunk 2]
```diff
@@ -1 +1,29 @@
...
+through them, you will learn about the setup of Fleet and its core functionalities.
+
+> Note
+>
+> All tutorials listed assume that you already have a basic understanding of Kubernetes. For more
+> information, see the [Kubernetes documentation](https://kubernetes.io/docs/home/).
+
+Fleet can work with many different Kubernetes environment setups. You may run Fleet on managed
+Kubernetes clusters, such as
+[Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/products/kubernetes-service)
```
Comment (ryanzhang-oss) [inline docs/getting-started/README.md:14]: does the solution work with GKE/EKS? If so then we should link those.

File: docs/getting-started/README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+or on on-premises Kubernetes clusters. It even works with local [kind](https://kind.sigs.k8s.io/)
```
Comment (ryanzhang-oss) [inline docs/getting-started/README.md:15]: nit: I think on premises cluster is a much more attractive value proposition than kind.

File: docs/getting-started/README.md [hunk 4]
```diff
@@ -1 +1,29 @@
...
+setup from the options below; through Fleet offers the same set of functionalities
+regardless of the setup environment, the actual setup process might be different across
+environments.
+
+* If you are familar with Microsoft Azure, or if you currently do not have a Kubernetes setup
+that you can run Fleet with, use the
+[Getting started with Fleet using Azure Fleet Manager](aks-fleet-manager.md) tutorial.
+
+    It is also a perfect option if you do not know which tutorial to begin with.
+
```
Comment (ryanzhang-oss) [inline docs/getting-started/README.md:26]: should this be moved to the aks specific doc?

File: docs/getting-started/aks-fleet-manager.md [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+In this tutorial, you will try Fleet out using
+[Azure Fleet Manager](https://azure.microsoft.com/en-us/products/kubernetes-fleet-manager),
+an Azure service that aims to help seamless manage Kubernetes clusters at scale.
+Azure Fleet Manager is built on top of the Fleet open-source project, which greatly simplifies
+the experience of setting things up while providing additional features; if you have
+deployed Azure Kubernetes Service (AKS) clusters before, or if you are
+familar with Azure services, this tutorial can be a perfect place for you to get started with
+Fleet. This tutorial is also a great option if you are less familar with Kubernetes
+administration tasks, or if you just would like a playground where you can play with
+the features Fleet offers, as Azure Fleet Manager can handle many setup tasks automatically
```
Comment (ryanzhang-oss) [inline docs/getting-started/aks-fleet-manager.md:12]: looks like this should belong to Azure doc?

**Review by ryanzhang-oss — COMMENTED** [review]
looks like the main part of the getting started should belong to Azure doc?

File: docs/getting-started/aks-fleet-manager.md [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+    ```sh
+    # Replace YOUR-SUBSCRIPTION-ID with the value of your own.
+    export SUBSCRIPTION_ID=YOUR-SUBSCRIPTION-ID
+    az account -s $SUBSCRIPTION_ID
+    ```
+
+* Install the Azure Fleet Manager CLI extension for your Azure CLI installation.
+
+    ```sh
+    az extension add --name fleet
```
Comment (circy9) [inline docs/getting-started/aks-fleet-manager.md:57]: For existing users, they need to use

az extension update --name fleet

**Comment by michaelawyu** [comment]
Closing as non-upstream docs are not kept in the repo.

**Pull request closed without merging**